templates/common/_base/units/crio.service.yaml

```diff
@@ -2,16 +2,7 @@ name: crio.service
 dropins:
   - name: 10-mco-default-env.conf
     contents: |
-      [Unit]
       {{if .Proxy -}}
-      [Service]
-      {{if .Proxy.HTTPProxy -}}
-      Environment=HTTP_PROXY={{.Proxy.HTTPProxy}}
-      {{end -}}
-      {{if .Proxy.HTTPSProxy -}}
-      Environment=HTTPS_PROXY={{.Proxy.HTTPSProxy}}
-      {{end -}}
-      {{if .Proxy.NoProxy -}}
-      Environment=NO_PROXY={{.Proxy.NoProxy}}
-      {{end -}}
+      [Unit]
+      EnvironmentFile=/etc/mco/proxy.env
```

Comment:
Systemd wants the `EnvironmentFile` declaration under the `[Service]` section.

Reply:
Fixed up.